Documentation~/ref_actions.md

```diff
@@ -1,17 +1,74 @@
 # Action reference
 
-This table lists all the actions available in ProBuilder, and indicates which [Edit modes](modes.md) support them.
+This table lists all the actions you can apply to a ProBuilder object, and indicates which [Edit modes](modes.md) support them. 
+
+These actions are available:
+
+* From the **main menu** > **Tools** > **ProBuilder**. The actions are grouped in **Selection**, **Interaction**, **Object**, and **Geometry**.
+* From the context menu: 
+    1. In the **Tools** overlay, set the working context to ProBuilder. 
+    1. In the **Tool Settings** overlay, select an edit mode.
+    1. In the **Scene** view, right-click on a ProBuilder object.
+
+<!--So... I divided this page by category. I think it's easier to scan. But then the question is: should the TOC match, or should the TOC stay alphabatised? -->
+
+## Selection actions
+
+These options are never available in the GameObject tool context.
+
+| **Property** | **Vertex edit mode (ProBuilder context)** | **Edge edit mode (ProBuilder context)** | **Face edit mode (ProBuilder context)** |
+| ---- | ---- | ---- | ---- |
+| [Grow Selection](Selection_Grow.md) | x | x | x |
+| [Select Hole](Selection_SelectHole.md) | x | x | |
+| [Select Loop (in Edge Selection mode)](Selection_Loop_Edge.md) | | x | |
+| [Select Loop (in Face Selection mode)](Selection_Loop_Face.md) | | | x |
+| [Select Material](Selection_SelectByMaterial.md) | | | x |
+| [Select Ring (in Edge Selection mode)](Selection_Ring_Edge.md) | | x | |
+| [Select Ring (in Face Selection mode)](Selection_Ring_Face.md) | | | x |
+| [Select Vertex Color](Selection_SelectByVertexColor.md) | x | x | x |
+| [Shrink Selection](Selection_Shrink.md) | | x | x | x |
+
+<!--two select loop should be one, two select ring should be one - that will have to be another ticket-->
```

Comment:
Don't forget to delete this and other comments.

Reply:
Ignoring this since other PRs killed this page anyway

Reply:
Oh, well - a future PR in a branch I'm already working on